pkg/agent/clean/adunmigration/ldap_test.go

```diff
@@ -0,0 +1,74 @@
+package adunmigration
+
+import (
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	logrusTest "github.com/sirupsen/logrus/hooks/test"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestEscapeUUID(t *testing.T) {
```

Comment:
Q: Do we want this and the next test to run in parallel by any chance? As it stands right now all other tests with `t.Parallel()` in this package will be paused until these two are finished.

Reply:
For `TestIsGUID` we can't use parallel because logrus state is global, and the hook assertion becomes a race condition. I'll add a note to clarify. For `TestEscapeUUID` I just missed it :)